content/push/index.textile

```diff
@@ -61,6 +61,26 @@ If you publish via channels, devices or browsers must subscribe to those channel
 
 * The "@clientId@":/push/publish#sub-clientID process subscribes all devices or browsers tied to a particular @clientId@ to a channel in one action, for example the same user's laptop and mobile phone. This approach is useful for subscribing multiple devices or browsers simultaneously.
 
+h2(#lifecycle). Push notification lifecycle
+
+The following diagram shows the push notification lifecycle:
+
+<a href="@content/diagrams/push-lifecycle.png" target="_blank">
+  <img src="@content/diagrams/push-lifecycle.png" style="width: 100%" alt="Push notifications lifecycle">
+</a>
+
+The following explains the push notification lifecycle:
+
+* Publish a push notification:
+** The process begins with your server, which publishes a push notification. This server could be any backend service that needs to send notifications to users' devices.
+** Your server sends the notification data to Ably, a realtime messaging platform.
```

Comment:
i think it reads a bit funny briefly explaining what ably is, anyone reading this should already be aware of this so i'd opt to omit this.

```suggestion
** Your server sends the notification data to Ably.
```